components/Statistics/OrgStatisticsTabs.js

```diff
@@ -1,41 +1,38 @@
 import React, { useEffect } from 'react'
 import { Tabs } from 'antd'
 import VTabs from '../VTheme/VTabs'
-import { connect } from 'react-redux'
 import { withOrgs } from '../../lib/redux/reduxApi.js'
-
 import ReduxLoading from '../Loading'
 import StatisticsPanel from './StatisticsPanel'
+import { useSelector, useDispatch } from 'react-redux'
+
+const OrgStatisticsTabs = (props) => {
+  const orgAdminFor = useSelector((state) => state.session.me.orgAdminFor)
+  const dispatch = useDispatch()
 
-export const OrgStatisticsTabs = (props) => {
   useEffect(() => {
-    props.dispatch(
+    dispatch(
       props.organisationsActions.get({
-        q: JSON.stringify({ _id: { $in: props.orgAdminFor } })
+        q: JSON.stringify({ _id: { $in: orgAdminFor } })
       })
     )
-  }, [props.orgAdminFor])
+  }, [orgAdminFor])
 
-  if (!props.organisations.sync) {
+  if (!props.organisations || !props.organisations.sync) {
     return <ReduxLoading entity={props.organisations} label='organisations' />
   }
 
-  const orgs = props.organisations.data
+  const orgs = props.organisations.data || []
```

Comment:
under what condition will the second part of this line be executed?  .data must be falsy an empty array is not false and .data should not be undefined.